src/main/java/models/Task.java

```diff
@@ -0,0 +1,65 @@
+package models;
+
+import java.io.Serializable;
+
+/**
+ * Task class that represents a Task that will be saved by Dub.
+ */
+public class Task implements Serializable {
+
+    /** Description of the task. */
+    protected String description;
+
+    /** Boolean value that shows whether the task is done or not. */
+    protected boolean isDone;
+
+    /**
+     * Constructor of the Task class.
+     *
+     * @param description Description of the Task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Make the Task done.
+     */
+    public void setDone() {
+        this.isDone = true;
+    }
+
+    /**
+     * Equals function that check whether 2 objects are equal or not.
```

Comment:
I'd prefer the method header starts with singular verb. See [this](https://se-education.org/guides/conventions/java/intermediate.html#comments).

So you can say: "Returns true if two objects are equal."